go.mod

```diff
@@ -3,6 +3,9 @@ module quortex.io/kubestitute
 go 1.16
 
 require (
+	github.com/Masterminds/goutils v1.1.1 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible
```

Comment:
You should probably use the last release (v3.2.1) : https://github.com/Masterminds/sprig/releases